src/utils/http.ts

```diff
@@ -0,0 +1,31 @@
+export const http = {
+    post(url: string, body: any) {
+        return fetch(new URL(url), {
+            method: "POST",
+            mode: "no-cors",
```

Comment:
Right now there is a problem with sending requests to `nwaku` REST API.
Since there is a `cors` problem present when browser sends `no-cors` then `Content-Type` is getting reset to `text/plain`. Because of this `nwaku` handled rejects requests. 

I think we can try approaching it by accepting `text/plain` as well and parsing body as `JSON`.